ReadIt/RootCoordinator.swift

```diff
@@ -29,11 +29,11 @@ class RootCoordinator: Coordinator {
 		let navigationController = RootNavigationController()
         navigationController.navigationBar.titleTextAttributes = [
 			.font: UIFont(name: "Palatino", size: 24)!,
-			.foregroundColor: UIColor.white
+			.foregroundColor: UIColor.invertedColors
```

Comment:
Do you really need that? Couldn't you use `.systemFill`?

Reply:
Make sense.